src/ui/BookSearch/RecentSearches.tsx

```diff
@@ -1,7 +1,11 @@
 import { Flex, Text, Box } from '@chakra-ui/react';
 
+interface searchedWordsProps {
+  keyword: string;
+  createdAt: string;
+}
 interface RecentSearchesProps {
-  searchedWords: string[];
+  searchedWords: searchedWordsProps[] | undefined;
```

Comment:
searchedWords를 optional type으로 선언해도 될 것 같아요.

```suggestion
  searchedWords?: searchedWordsProps[]
```